dashboard/app/views/devise/registrations/new.html.haml

```diff
@@ -1,91 +1,111 @@
           = f.text_field :name, class: 'input-xlarge fieldblock', autofocus: true, maxlength: 255
-        %br/
-        .itemblock
+          .error_in_field
+
+        #email-block.itemblock
           .labelblock
             = t('activerecord.attributes.user.email')
           = f.email_field :email, class: 'input-xlarge fieldblock', maxlength: 255
-        %br/
+          .error_in_field
+
         - if f.object.password_required?
-          .itemblock
+          #password-block.itemblock
             .labelblock
               = t('activerecord.attributes.user.password')
             = f.password_field :password, class: 'input-xlarge fieldblock', maxlength: 255
-            %span#password_message
-          %br/
-          .itemblock
+            .error_in_field
+
+          #password_confirmation-block.itemblock
             .labelblock
               = t('activerecord.attributes.user.password_confirmation')
             = f.password_field :password_confirmation, class: 'input-xlarge fieldblock', maxlength: 255
-          %span#password_message_confirmation
-        %br/
-        - if resource.teacher?
-          .itemblock
-            .labelblock
-              = t('signup_form.school_name')
-            = f.text_field :school, class: 'input-xlarge fieldblock', maxlength: 255
-          %br/
-          .itemblock
-            .labelblock
-              = t('signup_form.school_address')
-            = f.text_area :full_address, class: 'input-xxlarge fieldblock', placeholder: t('signup_form.address_placeholder'), rows: 4, maxlength: 1024
-          %br/
+            .error_in_field
 
-          = f.hidden_field :age, value: '21'
-        - if resource.student?
-          .itemblock
-            .labelblock
-              = t('signup_form.age')
-            = f.select :age, User::AGE_DROPDOWN_OPTIONS, include_blank: true, class: 'fieldblock'
-          %br/
-          .itemblock
-            .labelblock
-              = t('signup_form.gender')
-            = f.select :gender, gender_options
-          %br/
+        #schoolname-block.itemblock{style: "display: none"}
+          .labelblock
+            = t('signup_form.school_name')
+          = f.text_field :school, class: 'input-xlarge fieldblock', maxlength: 255
+          .error_in_field
+
+        #age-block.itemblock
+          .labelblock
+            = t('signup_form.age')
+          %select.fieldblock{name: "user[age]", type: "select", style: "width: 234px"}
+            %option{value: "", selected: true, disabled: true}
+            - User::AGE_DROPDOWN_OPTIONS.each do |age_option|
+              %option{value: age_option}= age_option
+          .error_in_field
+
+        #gender-block.itemblock
+          .labelblock
+            = t('signup_form.gender')
+          = f.select :gender, gender_options, { disabled: '', selected: '' }, { class: 'fieldblock', style: "width: 234px" }
+          .error_in_field
+
+        -# Not yet ready to turn this on.
```

Comment:
lol I know it's short term (we always think it is....) but maybe more descriptive?

`TODO (brendan): implement additional school info fields`